packages/grid/x-grid-data-generator/package.json

```diff
@@ -24,21 +24,19 @@
     "typescript": "tsc -p tsconfig.json"
   },
   "dependencies": {
+    "@material-ui/core": "^4.9.12 || ^5.0.0-alpha.22",
```

Comment:
It should be a peer dependency. There is a singleton.

Reply:
Done. I just wanted to know. What do you mean by `singleton`? Does it mean there is no version conflict?

Reply:
It means that it has little to no tolerance for duplication of the dependency in the bundle.